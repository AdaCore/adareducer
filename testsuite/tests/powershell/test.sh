$ADAREDUCER --single-file hello.adb p.gpr oracle.ps1 > /dev/null
cat hello.adb
