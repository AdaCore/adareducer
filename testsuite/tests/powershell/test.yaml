description: "powershell"
control:
  - [SKIP,
     "env.build.os.name != 'windows'",
     "This is a Powershell test, supported only on Windows"]

