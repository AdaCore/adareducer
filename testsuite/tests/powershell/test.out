procedure Hello is
begin
   Ada.Text_IO.Put_Line ("pouet");
end Hello;
